lms/djangoapps/teams/csv.py

```diff
@@ -170,22 +170,34 @@ def import_succeeded(self):
 
     def set_team_membership_from_csv(self, input_file):
         """
-        Assigns team membership based on the content of an uploaded CSV file.
+        Wrapper to parse CSV file to use for setting team memberships.
+        Returns true for success, false for failure
+        """
+        csv_data = csv.DictReader((line.decode('utf-8-sig').strip() for line in input_file.readlines()))
+        return self.set_team_memberships(csv_data)
+
+    def set_team_memberships(self, csv_data):
```

Comment:
maybe change `csv_data` to `csv_reader`? it is still a reader object.

Reply:
Updated variable name